source/_includes/installation/core.md

```diff
@@ -33,7 +33,7 @@ sudo apt-get upgrade -y
 Install the dependencies:
 
 ```bash
-sudo apt-get install -y python3 python3-dev python3-venv python3-pip libffi-dev libssl-dev libjpeg-dev zlib1g-dev autoconf build-essential libopenjp2-7 libtiff5 libturbojpeg0 tzdata
+sudo apt-get install -y python3 python3-dev python3-venv python3-pip libpython3.9-dev libffi-dev libssl-dev libjpeg-dev zlib1g-dev autoconf build-essential libopenjp2-7 libtiff5 libturbojpeg0-dev tzdata
```

Comment:
`python3-dev` already depends on `libpython3.9-dev`, see:

<https://packages.debian.org/bullseye/python3-dev>

The `libturbojpeg0-dev` change is unclear to me, as far as I'm aware we don't need to compile anything against it, so the non-dev version should be enough.